node/l1-public-v2.yml

```diff
@@ -48,14 +48,34 @@ paths:
               $ref: '#/components/schemas/Transaction'
       responses:
         200:
-          description: $DAG transaction has been received for processing
+          description: Hash of $DAG transaction that has been received for processing
           headers:
             X-Id:
               $ref: '#/components/headers/X-Id'
             X-Session-Token:
               $ref: '#/components/headers/X-Session-Token'
             Request-Signature:
               $ref: '#/components/headers/Request-Signature'
+          content:
+            application/json:
+              schema:
+                type: string
+        400:
+          description: $DAG transaction rejection reason
```

Comment:
Maybe "$DAG transaction is rejected"? As before, it's the description of the "result"

Reply:
If that's the concept, than of course, I will adjust it.